imod/mf6/model.py

```diff
@@ -495,17 +495,21 @@ def regrid_like(
                 raise ValidationError("\n" + status_info.to_string())
         return new_model
 
-    def _mask_all_packages(
+    def mask_all_packages(
         self,
-        domain: GridDataArray,
+        mask: GridDataArray,
     ):
         """
         This function applies a mask to all packages in a model. The mask must
         be presented as an idomain-like integer array that has 0 or negative
         values in filtered cells and positive values in active cells
         """
+        if any([d not in ["x", "y", "layer", "mesh2d_nFaces", "dx", "dy"] for d in mask.coords]):
```

Comment:
what is d? Would 'coor', 'coordinate' or 'dimension' be a better name?

Reply:
Xarray would use ``coord`` here,  I think that is the right naming.